MinecraftClient/Protocol/Handlers/PacketOutgoingType.cs

```diff
@@ -19,6 +19,10 @@ enum PacketOutgoingType
         TabComplete,
         PlayerPosition,
         PlayerPositionAndLook,
-        TeleportConfirm
+        TeleportConfirm,
+        //modified by reinforce
```

Comment:
There's information about who added what in the git log and git blame sections, adding too many of these would make the code less readable in the long run. You can use `<remarks>added by reinforce</remarks>` instead in method descriptions, if needed.